docs/LSPSupport.md

```diff
@@ -189,7 +193,14 @@ Here is an example with the [Qute language server](https://github.com/redhat-dev
 
 ![textDocument/completion](./images/lsp-support/textDocument_completion.png)
 
-The [completionItem/resolve](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#completionItem_resolve) is implemented too.
+The [completionItem/resolve](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#completionItem_resolve) is implemented to resolve the property:

```

Comment:
The [completionItem/resolve] request

Reply:
is implemented to resolve:

* the `documentation` property of a completionItem